court_ball_game/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from court.models import Court
+from player.models import BallGame
+
+MAX_LEN_BALL_GAME_STR = 10
+
+
+class CourtBallGame(models.Model):
```

Comment:
Do we need the unique together stuff that we have for event player?

Reply:
No, I tryed to merge t because I had some conflicts in settings.py file

Reply:
Don't know if this comment belongs here, but you should solve conflicts with rebase not merge. If you need help solving the current issue send out a shout.

Reply:
Back to the original comment, I think we need the unique together stuff here. Similarly to the event player module.

Reply:
Done.

Reply:
I don't see the unique_together stuff, not the test for it.